### PR TITLE
add link to resource and add link to others

### PR DESCRIPTION
**Pull request by rodmoioliveira**
Hi @matiasmascioto. 

Thank you for gathering all these references. Amazing work!
I add some stuff. Best regards!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ A curated list of awesome resources related to *Soccer Analytics* in *english* a
 
 ## Data
 ### Datasets
-* [A public data set of spatio-temporal match events in soccer competitions]
+* [A public data set of spatio-temporal match events in soccer competitions](https://doi.org/11.6084/m9.figshare.c.4415000) : Free - .json - UEFA Euro Cup 2016 - La Liga, Serie A, Bundesliga, Premier League, Ligue 1, FIFA World Cup 2018
```
Comment (matiasmascioto) [inline README.md]: The link doesn't work.

**Review by matiasmascioto — COMMENTED** [review]
Hi! Thank you for contributing. Please check out one of the links.

File: README.md (hunk 1 above)
Comment (rodmoioliveira) [reply README.md]: Sorry, try again: 

https://figshare.com/collections/Soccer_match_event_dataset/4415000

File: README.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ A curated list of awesome resources related to *Soccer Analytics* in *english* a
 
 ## Data
 ### Datasets
-* [A public data set of spatio-temporal match events in soccer competitions](https://doi.org/11.6084/m9.figshare.c.4415000) : Free - .json - UEFA Euro Cup 2016 - La Liga, Serie A, Bundesliga, Premier League, Ligue 1, FIFA World Cup 2018
+* [A public data set of spatio-temporal match events in soccer competitions](https://figshare.com/collections/Soccer_match_event_dataset/4415000) : Free - .json - UEFA Euro Cup 2016 - La Liga, Serie A, Bundesliga, Premier League, Ligue 1, FIFA World Cup 2018
```
Comment (rodmoioliveira) [inline README.md:81]: Fix it!

**Pull request merged**